docs/iex.md

```diff
@@ -0,0 +1,70 @@
+# TorQ-IEX
+
+An example production ready market data capture system, using randomly generated financial data along with market data pulled from the IEX. This is installed on top of the base TorQ package, and includes a version of [kdb+tick](http://code.kx.com/wsvn/code/kx/kdb+tick).
```

Comment:
Is this first sentence correct?  This has been added to the FSP now so I think this wording needs changed